swagger/scylla-manager.json

```diff
@@ -569,8 +569,8 @@
             "type": "string"
           }
         },
-        "retention": {
-          "type": "integer"
+        "retention_policy": {
+          "$ref": "#/definitions/RetentionPolicy"
```

Comment:
Additionally, this change should be announced in changelog on the new release of manager. Nothing that requires your action now, but we cannot forget about it.

Reply:
Good catch!
Actually we can just keep it in the same way as before. The important part was changing `int -> *int` for `retention` and `retention_days` here:
```go
// taskProperties is the main data structure of the runner.Properties blob.
type taskProperties struct {
	Keyspace         []string     `json:"keyspace"`
	DC               []string     `json:"dc"`
	Location         []Location   `json:"location"`
	Retention        *int         `json:"retention"`
	RetentionDays    *int         `json:"retention_days"`
	RetentionMap     RetentionMap `json:"retention_map"`
```
in order to properly handle default values. I believe that this change isn't breaking compatibility, because we are not returning it to user at any point